[Content] Remnant Job To Disable Worldship (#7698)
endless-sky/endless-sky: data/remnant/remnant jobs.txt

	on visit
		dialog
			`You have returned, but there is still no word from the missing ship.`

mission "Remnant: Disable 1"
	job
	repeat
	name "Disable Damaged Korath World Ship"
	description "A damaged Korath world ship is stuck in the Ember Waste sectors boarding Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. After this return to <planet> to receive your payment of <payment>."
	source
		government "Remnant"
	to offer
		has "Remnant: Wont Someone Please Think Of The Children: done"
		random < 5
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system 
			near "Parca" 1

> 		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"

Reviewer: We shouldn't use a constant name for a repeatable job...

```suggestion
		ship "Kas'lor Ik 582 (Crippled)"
```

Reviewer 2: ```suggestion
		fleet
			names "korath"
			variant
				"Ra'gru Ik 618 (Crippled)"
```
like this then, the way you had done it it would have no name

	on visit
		dialog
			`You have returned, but there is still no word from the missing ship.`

mission "Remnant: Disable 1"
	job
	repeat
	name "Disable Damaged Kas'lor Ik 582"
	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
	source
		government "Remnant"

> 	to offer
> 		has "Remnant: Will Not Someone Please Think Of The Children: done"

Reviewer: It says below if you abort or fail that it will not be offered again, but it'll offer again anyway.

```suggestion
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		not "Remnant: Disable 1: aborted"
		not "Remnant: Disable 1: failed"
```

Reviewer 2: I'm still dubious on if we should offer the WS on a plate (its defenceless)and then not offer it again, caus most ppl would just cap it.
My idea when proposing the mission was to encourage looting but I wouldnt mind capping it I guess (also it being in postverta means its locked until we unlock it and that exiles know of it)

Reviewer 2: For context, ppl were willing to halt the entire remnant story to get a garanteed WS. Granted this one is cargo and not crew but still.

Author: The 582 can now be captured and extracted from Korath space. So I think this is an ok trade off.

	on visit
		dialog
			`You have returned, but there is still no word from the missing ship.`

mission "Remnant: Disable 1"
	job
	repeat
	name "Disable Damaged Kas'lor Ik 582"
	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
	source
		government "Remnant"
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		random < 5
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system 
			near "Antevorta" 1
		fleet
			names "korath"
			variant

> 				"Kas'lor Ik 582 (Crippled)"

Reviewer 2: Just thinking, we could maybe have another variant? 
Should prob be done in another PR after this one is merged though. I'm thinking a 577 or a 590?

Author: 582 was selected for these reasons outlined by @ravenshining 
https://github.com/endless-sky/endless-sky/pull/7689#pullrequestreview-1193000780

TLDR Effectively of the civilian ships only 582 should be seen outside the Korath core sectors. And then it should be assumed that it is on some type of refueling mission.

		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
	on visit
		dialog "You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning."
	on abort
		"reputation: Remnant" += 250
	on fail
		"reputation: Remnant" -= 250

> 		dialog
> 			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`

Reviewer 2: I just realized you would get this dialog upon aborting the mission...
```suggestion
		conversation
			branch failed
				not "Remnant: Disable 1: aborted"
			action
				clear "Remnant: Disable 1: aborted"
			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`
				decline
			label failed
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

	on complete
		"remnant: disable and save count" ++
		payment 2000000

> 		dialog "A Remnant military leader thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon payment of <payment>."

Reviewer 3: ```suggestion
		dialog "A Remnant military prefect thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon logistics adjustment of <payment>."
```

While basically being payment by another name, it is good to stick with the "logistics adjustment" framework to maintain consistency. (As in, they aren't *paying* the player, they are providing them with an adjustment to the amount of logistical support they can obtain, as required to deal with damage and expenses incurred in the process of the mission.)

	on visit
		dialog
			`You have returned, but there is still no word from the missing ship.`

mission "Remnant: Disable 1"
	job
	repeat
	name "Disable Damaged Kas'lor Ik 582"

> 	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."

Reviewer 4: `A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space`
This isn't really correct. Postverta and co. aren't bordering on Korath space.
```suggestion
	description "A damaged Kas'lor Ik 582 is stuck in a newly accessible system, near Postverta. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
```

	source
		government "Remnant"
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		not "Remnant: Disable 1: aborted"
		not "Remnant: Disable 1: failed"
		random < 5
	npc disable save
		government "Korath"

> 		personality heroic vindictive target marked staying

Reviewer: ```suggestion
		personality heroic target marked staying disables plunders
```

		system 

> 			near "Vaticanus" 1

Reviewer 4: Just to confirm, this covers Vaticanus, Egeria, and Prosa?
Those systems and those systems only are where you want this to spawn?

Author: Basically it needs to spawn in that cluster of sectors but not in sector with the station. This was the easiest way I could think of.

		fleet
			names "korath"
			variant
				"Kas'lor Ik 582 (Stranded)"

> 		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`

Reviewer 2: just checked it doesnt work that way, I wish it did though

> 		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`

Reviewer 5: This problem of wanting the same random location used by multiple fleets could be solved by #1934.

> 		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`

Reviewer 5: ```suggestion
		dialog `You have disabled the Kas'lor Ik 582. Time to report back to <planet>.`
```

	on visit
		dialog `You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning.`
	on abort

> 		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`

Reviewer 3: ```suggestion
		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They advise you they will prioritize other teams for these missions in the future.`
```

	on visit
		dialog
			`You have returned, but there is still no word from the missing ship.`

mission "Remnant: Disable 1"
	job
	repeat
	name "Disable Damaged Kas'lor Ik 582"
	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
	source
		government "Remnant"
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		random < 5
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system 
			near "Vaticanus" 1
		fleet
			names "korath"
			variant
				"Kas'lor Ik 582 (Stranded)"
		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
	on visit
		dialog "You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning."
	on abort
		"reputation: Remnant" += 250
	on fail
		"reputation: Remnant" -= 250
		conversation
			branch failed
				not "Remnant: Disable 1: aborted"
			action
				clear "Remnant: Disable 1: aborted"

> 			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`

Reviewer 4: `remnants`?
Should probably start with a capital and perhaps not have an `s` at the end.

				decline
			label failed

> 			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`

Reviewer 4: ```suggestion
			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```
